package.json

```diff
@@ -49,10 +49,8 @@
     "parse:area": "tsc && node -r dotenv/config ./build/parse:area",
     "parse:timeline": "tsc && node -r dotenv/config ./build/parse:timeline",
     "build": "tsc",
-    "start:watch": "tsc-watch --onSuccess \"node -r dotenv/config ./build/app.js\"",
-    "start": "node -r dotenv/config ./build/app.js",
+    "start": "tsc-watch --onSuccess \"node -r dotenv/config ./build/app.js\" --inspect=0.0.0.0",
```

Comment:
i think you need to do it in other way around
`node -r dotenv/config ./build/app.js --inspect=0.0.0.0`